llvm/test/CodeGen/RISCV/GlobalISel/irtranslator/calling-conv-ilp32-ilp32f-ilp32d-common.ll

```diff
@@ -16,6 +16,559 @@
 ; Check that on RV32, i64 is passed in a pair of registers. Unlike
 ; the convention for varargs, this need not be an aligned pair.
 
+define i64 @callee_128i_in_regs_stack_fst(i64 %x1, i64 %x2, i64 %x3, i64 %x4, i128 %y ) {
```

Comment:
Please add one more i128 argument (here and in the caller), there should be difference in alignments of the stored pointers.

Reply:
Added that argument.

Reply:
Thanks. Just in case, is i12**7** a typo or intentional?